proseco/tests/conftest.py

```diff
@@ -7,6 +7,11 @@ def use_fixed_chandra_models(monkeypatch):
     monkeypatch.setenv("CHANDRA_MODELS_DEFAULT_VERSION", "3.48")
 
 
+@pytest.fixture(autouse=True)
```

Comment:
Can you try without `autouse=True` and see how many tests fail? I'm getting less keen on the idea of our tests defaulting to a non-flight behavior for everything.

Reply:
It is a subtlety I'd missed that basically all the tests pass without this set now that we've moved to the tri-state on PROSECO_ENABLE_FID_OFFSET and our standard get_aca_catalog test setup doesn't actually set t_ccd_acq https://github.com/sot/proseco/blob/85c79bef6f1b7ef9594706d23743a1b947b3a08f/proseco/tests/test_common.py#L14 .  I think we should probably update our standard regression tests to call get_aca_catalog the way it is used in operations.